admin/sudo/patches/010-cross-compile-fixes.patch

```diff
@@ -13,4 +12,4 @@ diff -rupN sudo-1.8.11p2.orig/lib/util/Makefile.in sudo-1.8.11p2/lib/util/Makefi
 +	mksigname > $@
  
  mksiglist: $(srcdir)/mksiglist.c $(srcdir)/mksiglist.h $(incdir)/sudo_compat.h $(top_builddir)/config.h
- 	$(CC) $(CPPFLAGS) $(CFLAGS) $(DEFS) $(srcdir)/mksiglist.c -o $@
```

Comment:
Why is `$(DEFS)` no longer needed?  The commit message doesn't mention this.

Reply:
You're not reading it right. That's a part of the stock file, not what is being patched

Reply:
I can confirm this is correct, $(DEFS) was removed from the original file a few releases back but the patch has not been updated to reflect this change.